tests/migration/migration.go

```diff
@@ -3140,6 +3141,196 @@ var _ = SIGMigrationDescribe("VM Live Migration", decorators.RequiresTwoSchedula
 			Eventually(matcher.ThisMigration(migration)).WithPolling(1 * time.Second).WithTimeout(60 * time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceMigrationRejectedByResourceQuota))
 		})
 	})
+	Context("Virtiofs", decorators.VirtioFS, func() {
+		DescribeTable("should migrate with a shared", func(createConfigVolume func() (string, string, string, []libvmi.Option)) {
+
+			configVolumeName, expectedOutput, testCommand, configVolumeOption := createConfigVolume()
+
+			vmi := libvmifact.NewAlpine(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			)
```

Comment:
```suggestion
				libvmi.WithNetwork(v1.DefaultPodNetwork()),
				configVolumeOption...,
			)
```

This might work

Reply:
it should, but it doesn't.... go so silly language :)

(This test will be removed)